sdk/spring-sdk/src/main/java/kalix/springsdk/annotations/EntityType.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package kalix.springsdk.annotations;
+
+import java.lang.annotation.*;
+
+/**
+ * Assign a type to the entity. This name should be unique among the different existing entities
+ * within a Kalix application. Additionally, the name should be stable as a different name means a
+ * different entity in storage.
```

Comment:
Would it be worth explaining why? What would happen if the name is not stable?

Reply:
It says that a different name is a different entity in storage. Do you mean that we should be more explicit? 

I will try to find a better text. Stay tuned...